arviz_dashboard/__init__.py

```diff
@@ -1,4 +1,5 @@
 """Bayesian Dashboards built on top of Panel"""
 __version__ = "0.0.1a"
 
-from .elpd import dashboard_elpd
\ No newline at end of file
+from .elpd import dashboard_elpd
+from .ppc import dashboard_ppc
```

Comment:
Not necessary to change now, but we should think of better names for the dashboard functions. If we are going to import `arviz_dashboard as azd` or something similar then we could omit the dashboard from the names.

Reply:
Yeah,I do think we should think another name for this. dashboard_elpd sames too long